aion/web/controllers/answer_controller.ex

```diff
@@ -52,4 +54,10 @@ defmodule Aion.AnswerController do
 
     send_resp(conn, :no_content, "")
   end
+
+  def unauthenticated(conn, _params) do
+    conn
+    |> put_status(401)
+    |> render("error.json", message: "Authentication required")
```

Comment:
I agree but I would prefer to put it in authorization related module

Reply:
This is going to be a one function only. I think it could be better if we just stored all status-related things used across all controllers in one file around the controllers. There is a reason why the unauthenticated function lies in the controller implementation and I guess anyone from outside the project trying to see where they get a certain status would look in the controller area. 

I would agree to put it in auth area if it was auth only thing, but there are also other error statuses and splitting them to be in different areas would make it harder to find certain implementations. We're going to have to create the `errors.ex` or something like that anyway for cases like `unprocessable entity` etc. Unless there is a module in authentication area where this function would fit in perfectly, I would opt for putting all common error rendering functions in one module.